code/modules/research/nanites/nanite_programs/protocols.dm

```diff
@@ -0,0 +1,103 @@
+//Replication Protocols
+/datum/nanite_program/protocol/kickstart
+	name = "Kickstart Protocol"
+	desc = "Replication Protocol: the nanites focus on early growth, heavily boosting replication rate for a few minutes after the initial implantation."
+	use_rate = 0
+	rogue_types = list(/datum/nanite_program/necrotic)
+	protocol_class = NANITE_PROTOCOL_REPLICATION
+	var/implantation_time = 0
+	var/boost_duration = 1200
+
+/datum/nanite_program/protocol/kickstart/on_mob_add()
```

Comment:
can you delete the kickstart program and then reinstall it (using a killcode, some repeaters, and cloud syncing) every minute or so to keep this bonus basically constantly active?

Reply:
ah, my bad, i mixed it up with the nanites' proc. I'll make sure it's based on nanite implantation time and not program installation time